### ISSUE TITLE
rename v2 to v1

### ISSUE THREAD

**Issue by beckermr** [conda-forge/conda-smithy#2030]
> I think I've said this before, but I call them V0 and V1 recipes. The original recipe format doesn't have a schema_version field and the new format has schema_version set to 1. I'm sure a lot of people will get confused if V2 implies a value of 1.

As @schuylermartin45 points out, v2 is really v1. We should rename again in the code. 

Cc @wolf @isuruf @nichmor

**Comment by wolfv** [comment]
"insert table flip emoji" :D 

From my POV I don't really care. Happy to call it V1 or V2, it doesn't matter. I do think, given that "meta.yaml" has been around for quite some time, it's fair to call it V1 but again, really don't care much.

**Comment by beckermr** [comment]
I agree on the tableflip. This GIF is useful

https://media.giphy.com/media/X83Y7r03T6uty/source.gif

I am happy to do the rename myself, but I wanted to discuss first.

**Comment by beckermr** [comment]
The issue is that the schema_version in the new format is 1, not 2, which is massively confusing to folks, including me, now that I know this and am reading the code. 

We voted on and adopted schema_version=1 as a community and so now we have to live with it or do more voting.

**Comment by beckermr** [comment]
@wolfv This task is blocked on https://github.com/conda-forge/conda-smithy/pull/2029. Let's get that in first so we can rename all at once.

**Comment by wolfv** [comment]
I already renamed things in that PR :D

**Comment by schuylermartin45** [comment]
Sorry for sparking this and not bringing it up in such a public way much sooner. But I do think this will solve a lot of future headaches.

I think as for Wolf's marketing concerns, V0 sounds beta-y and V1 sounds like a fully realized product. I think the gravitas of the shift isn't lost. Besides, we should all be pretty comfortable with counting from 0. 😜 

I've been using this nomenclature internally and in Conda Recipe Manager for some time now. I'm just happy I eventually moved from "old recipe format" and "new recipe format"....that would have been so much worse to deal with later.

**Comment by jezdez** [comment]
> We voted on and adopted schema_version=1 as a community and so now we have to live with it or do more voting.

Y'all, make sure the CEP is accurate please.

**Comment by jezdez** [comment]
As a side-note, I'm a little concerned we have standard conversations outside the cep repo, let's open issues about existing and voted on CEPs there next time?

**Comment by jezdez** [comment]
I opened https://github.com/conda/ceps/issues/89

**Comment by beckermr** [comment]
Agreed @jezdez. CEP-14 clearly says what the version is here: https://github.com/conda/ceps/blob/main/cep-14.md#schema-version. The convo above was ensuring that we adopt the same language.
